packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -84,7 +84,7 @@ const DetailComponent = ({
                     pageType,
                     expirationDate: expiration_date,
                     documentId: data.document_id || '',
-                    lifetimeValid: +(lifetime_valid && !expiration_date),
+                    lifetimeValid: lifetime_valid && !expiration_date,
```

Comment:
`lifetimeValid` takes a numerical value 0 or 1. But this change makes it true or false. Is this expected?

Reply:
Thanks for noticing, will check